helm-chart-sources/diego-core/templates/diego-application-controller-crds/management.diegocloud.io_environments.yaml

```diff
@@ -89,12 +89,20 @@ spec:
                           type: boolean
                         synced:
                           type: boolean
+                       versionRef:
++                          properties:
```

Comment:
not the plusses!

Reply:
thanks, believe I have fixed this, can you re-check pls.